src/index.js

```diff
@@ -31,9 +31,31 @@ export default function routing(config: Config) {
     return config;
   };
 
+  const getParams = function(path: string): Object {
+    const pattern = Object.keys(config).find(patternKey =>
+      pathToRegexp(config[patternKey]).exec(path),
+    );
+
+    if (!pattern) {
+      return {};
+    }
+
+    const paramKeys = [];
+    const paramsArray = pathToRegexp(config[pattern], paramKeys).exec(path);
```

Comment:
Is this not already compiled on the paths object?

Reply:
the `paths` object contains methods that use `pathToRegExp.compile()` which turns an object into a path i.e. the wrong way round.